lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -43,14 +44,13 @@ def default_key_transform
 
       # {http://jsonapi.org/format/#crud Requests are transactional, i.e. success or failure}
       # {http://jsonapi.org/format/#document-top-level data and errors MUST NOT coexist in the same document.}
-      def serializable_hash(options = nil)
-        options ||= {}
+      def serializable_hash(_options = nil)
```

Comment:
So, we do need to take options into `serializable_hash from the renderer. I'm willing to be convinced, but when someone passing in options from the `ActiveModel::Serializers::JSON` API to `as_json`, I would expect them to be honored, rather than silently ignored.